gsa/src/web/entities/table.js

```diff
@@ -270,7 +270,11 @@ class EntitiesTable extends React.Component {
         <UpdatingStripedTable
           header={header}
           footer={footer}
-          isUpdating={isUpdating}
+          isUpdating={
+            isDefined(entities) && entities[0].entityType === 'task'
```

Comment:
isArray is from gmp/utils/identity and getEntityType from gmp/utils/entitytpe
```suggestion
            isArray(entities) && getEntityType(entities[0]) === 'task'
```

Reply:
Done!